### PR TITLE
Add plugins

### PR DESCRIPTION
**Pull request by OliverNocon**
some of these plugins are actively used by existing `piper-lib-os` steps
other plugins are used in existing scenarios which should use this image further down the road

### CODE REVIEW

**Comment by benhei** [comment]
Are those plugins directly used by Piper 3 pipelines or only in specific scenarios?
I would aim at shipping only a bare minimum that is necessary for successfully running pipeline templates.

File: jenkins-master/plugins.txt [hunk 1]
```diff
@@ -12,32 +15,43 @@ credentials-binding
...
+extended-read-permission
 findbugs
 gatling
 ghprb
 github-branch-source
 greenballs
+groovy-postbuild
 htmlpublisher
 http_request
+influxdb:1.20.1
```
Comment (CCFenner) [inline jenkins-master/plugins.txt]: Most recent (1.20.4) should support SonarQube LTS again, needs to be verified.

File: jenkins-master/plugins.txt [hunk 2]
```diff
@@ -12,32 +15,43 @@ credentials-binding
 cucumber-testresult-plugin
 docker-workflow
 email-ext
+embeddable-build-status
```
Comment (CCFenner) [inline jenkins-master/plugins.txt]: *nice-to-have*

File: jenkins-master/plugins.txt [hunk 3]
```diff
@@ -12,32 +15,43 @@ credentials-binding
...
 junit
 kubernetes
 lockable-resources
 matrix-auth
 nexus-artifact-uploader
+Parameterized-Remote-Trigger
 parameterized-trigger
 performance
 pipeline-build-step
+pipeline-githubnotify-step
```
Comment (CCFenner) [inline jenkins-master/plugins.txt]: *nice-to-have*

**Comment by OliverNocon** [comment]
I removed the 'nice to have' ones

**Comment by fwilhe** [comment]
Hi @OliverNocon, If this is still relevant, please open a new pr in https://github.com/SAP/devops-docker-cx-server

**Comment by OliverNocon** [comment]
sure, it is. nobody approved this one though ...

**Pull request closed without merging**